modin/pandas/test/test_groupby.py

```diff
@@ -15,6 +15,10 @@
 import itertools
 import pandas
 import numpy as np
+from modin.config.envvars import Engine
+from modin.core.dataframe.pandas.partitioning.axis_partition import (
```

Comment:
Do we really want the test layer to depend on internals?

Reply:
I think it's okay to depend on internals here. In fact, I'm surprised there isn't already a test that depends on the Engine used (given we are not feature complete between Ray and Dask).

Of course, a better solution to this would be to unit test the internals, not test internals in API-level unit tests. What do you think? @YarShev ?

Reply:
I don't think that is a good idea to test each functionality (layer) separately. As a rule, tests should check all functionality by going top down, i.e. starting to call the API layer.

Although, I would prefer to get rid of the internals usage in tests in the future but now let's leave as is.